Tests/Services/OpenSSLKeyLoaderTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Tests\Services;
+
+use Lexik\Bundle\JWTAuthenticationBundle\Services\OpenSSLKeyLoader;
+
+/**
+ * OpenSSLKeyLoaderTest
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class OpenSSLKeyLoaderTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var OpenSSLKeyLoader */
+    protected $keyLoader;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp()
+    {
+        $this->keyLoader = new OpenSSLKeyLoader('private.pem', 'public.pem', 'foobar');
+    }
+
+    /**
+     * Test load unreadable public key.
+     *
+     * @expectedException        RuntimeException
+     * @expectedExceptionMessage Public key "public.pem" does not exist or is not readable.
+     */
+    public function testLoadUnreadablePublicKey()
+    {
+        $this->keyLoader->loadKey('public');
+    }
+
+    /**
+     * Test load unreadable private key.
+     *
+     * @expectedException        RuntimeException
+     * @expectedExceptionMessage Private key "private.pem" does not exist or is not readable.
+     */
+    public function testLoadUnreadablePrivateKey()
+    {
+        $this->keyLoader->loadKey('private');
+    }
+
+    /**
+     * Test load unreadable private key.
```

Comment:
Should be `* Test load invalid private key.`, same for the next but public.
It would be nice if you can fix it in a next commit @slashfan. 
Thanks by advance!